test/test/helpers/use-filesystem-mocks/index.js

```diff
@@ -27,6 +27,6 @@ function findQuickMocks( dirpath ) {
 	} );
 }
 
-export default function( dirpath ) {
+export default function( dirpath = __dirname ) {
```

Comment:
isn't this the path of this file though? I didn't provide a default because I think it's always wrong... but maybe I'm wrong?